utils/hct/hctbuild.cmd

```diff
@@ -47,7 +47,7 @@ set INSTALL_DIR=
 set DEFAULT_EXEC_ADAPTER=-DTAEF_EXEC_ADAPTER=
 set LIT_ARGS=
 set FRESH=
-set NUGET_WARP_EXTRA_ARGS=
+set NUGET_CONFIG_FILE=
```

Comment:
If you hadn't changed this line then I don't think you'd need to mess around with the `if defined` stuff around line 218.